### PR TITLE
Mai get involved sidebar 2nd pull

### PR DESCRIPTION
**Pull request by shmaiii**
## Description

Add sidebar to the Get Involved page

## Updates

[Please enter a brief summary of updates to this PR during code review - if any]

## Checklist

- [ ] Ran `npm run lint:fix` to format code
- [ ] Requested at least one reviewer
- [ ] Connected PR to Trello ticket (steps below)
- [ ] Addressed code review comments
- [ ] Gained at least one approval for your PR

## Connecting your PR to the corresponding ticket:
- Click on the "GitHub" button found on the right of your Trello ticket
- Choose "Attach PR" from the dropdown list
- Link your GitHub account (if you haven't done so already)
- Navigate to`VCL-content-platform` and choose your PR from the dropdown.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -66,7 +66,6 @@
     "@types/react-burger-menu": "^2.8.3",
     "@typescript-eslint/eslint-plugin": "^5.38.0",
     "@typescript-eslint/parser": "^5.38.0",
-    "eslint": "^8.23.1",
```
Comment (ElisaHu1) [inline package.json:69]: is there a reason you want to delete this package?

File: src/components/GetInvolvedSidebar/index.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export {default} from './GetInvolvedSidebar';
```
Comment (ElisaHu1) [inline src/components/GetInvolvedSidebar/index.ts:1]: you should add your getinvolve sidebar to the navbar. 
i cannot see anything from here 
image

File: package.json (hunk 1 above)
Comment (shmaiii) [reply package.json:69]: no i think it was either not there when I cloned it, or it was deleted when I deleted package.json and ran npm install again to fix the errors. I will add it back

File: src/components/GetInvolvedSidebar/index.ts (hunk 2 above)
Comment (shmaiii) [reply src/components/GetInvolvedSidebar/index.ts:1]: Am I supposed to add it to the navbar? I thought I should add it to the Get Involved page but it's not available right now so I only made one to test and then deleted it. Am I supposed to also create a file for the Get Involved page as well?

**Review by svolterra — COMMENTED** [review]
The styling of the sidebar looks good to me! However, I would rather we wait until this [Get Involved page PR](https://github.com/UBC-VCL/VCL-content-platform/pull/60) is merged before we review and merge this one 🤔 Just to see how the two work together :) Nice work!!! 😄🌟

**Comment by ElisaHu1** [comment]
@shmaiii Cathy has her change merged in Main, could you rebase your branch on top of hers?

**Comment by shmaiii** [comment]
Do you mean rebase the branch onto main or onto cathy's branch?

**Comment by ElisaHu1** [comment]
i have merged cathy's branch to main, so main

**Comment by shmaiii** [comment]
okay i think i just did it.. Does it work?

**Comment by svolterra** [comment]
@shmaiii sorry it doesn't seem to be working right now on my end! Will try to see why :)

**Comment by ElisaHu1** [comment]
does not work for me either @shmaiii

**Review by ElisaHu1 — APPROVED** [review]

**Pull request merged**